services/graph/pkg/service/v0/educationclasses.go

```diff
@@ -166,8 +176,8 @@ func (g Graph) PatchEducationClass(w http.ResponseWriter, r *http.Request) {
 		}
 		return
 	}
-	render.Status(r, http.StatusNoContent) // TODO StatusNoContent when prefer=minimal is used, otherwise OK and the resource in the body
-	render.NoContent(w, r)
+	render.Status(r, http.StatusOK) // TODO StatusNoContent when prefer=minimal is used, otherwise OK and the resource in the body
```

Comment:
I don't think we should change that here. In this PR. We have a couple of other places where we still return 204 without the `prefer=minimal` set. If we want to change that we should make that consistently everywhere. But I think that should be part of a separate PR. Would you mind adding an issue for that?

Reply:
Reverted the lines and added the issue here: https://github.com/owncloud/ocis/issues/5445